equinix/resource_fabric_service_profile_acc_test.go

```diff
@@ -17,139 +17,134 @@ import (
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 )
 
-func testAccFabricReadServiceProfileConfig(uuid string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profile" "test" {
-	uuid = "%s"
-	}
-`, uuid)
-}
-
-func testAccFabricReadServiceProfilesListConfig(name string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profiles" "test" {
-		filter {
-			property = "/name"
-			operator = "="
-			values = ["%s"]
-		}
-	}
-`, name)
-}
-
-func TestAccFabricReadServiceProfileByUuid(t *testing.T) {
+func TestAccFabricCreateServiceProfile_SP_FCR_ba(t *testing.T) {
+	spName1 := "fabric_tf_acc_test_01"
```

Comment:
Our resource names should end in `_PFCR` so that if we need to sweep them up (i.e. delete them because the Unit Tests didn't for some reason) then we can.

Reply:
The spName's don't need to have they're own variable string. You could put them directly in the method signature and test assertions.

Reply:
done